app/assets/stylesheets/replies.scss

```diff
@@ -482,7 +482,11 @@ div.post-subheader { width: 100%; }
   font-size: 70%;
 }
 
-.reply-highlighted { border: 1px #FF0000 solid; }
+.reply-highlighted {
+  border-width: 1px;
+  border-style: solid;
+  border-color: $border_color_unread_reply;
```

Comment:
Why are these split out into multiple rules?

Reply:
Good question! I think it might have confused me or something at the time.
Reverted now.